components/RuiButton/readme.mdx

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
@jennasalau Should all the readme files be updated to `readme.mdx` as well? Or is the extension a typo

Reply:
No good point, lets make sure they are all .md so that github finds and renders them